default.nix

```diff
@@ -10,6 +10,7 @@ in
 , forceDontCheck ? false
 , enableProfiling ? false
 , enableDebugging ? false
+, enableBenchmarks ? true
```

Comment:
You asked whether this is a good default. Yes, it makes sense. Devs should get feedback about benchmarks that don't build correctly, but we shouldn't spend time running all the benchmarks every build by default.